src/ext/themelist.js

```diff
@@ -24,6 +24,7 @@ var themeData = [
     ["Kuroir"],
     ["KatzenMilch"],
     ["SQL Server"           ,"sqlserver"               , "light"],
+    ["CloudEditor", "cloud_editor", "light"],
```

Comment:
**nitpick:** Could you use the same formatting convention that's used for others? I.e., aligning indexes vertically